tests/util/block.ts

```diff
@@ -219,9 +221,44 @@ export const verifyBlockFees = async (
                 txBurnt += tipFeePortions.burnt;
               } else {
                 // For a regular substrate tx, we use the partialFee
-                let feePortions = calculateFeePortions(fee.partialFee.toBigInt());
-                txFees = fee.partialFee.toBigInt();
-                txBurnt += feePortions.burnt;
+                console.log(`evaluating substrate fee: ${fee}`);
```

Comment:
Shouldn't this be "debug" instead of console.log?

Reply:
Heh, sorry. Removed in 316c698